### PR TITLE
DDCE-1409 Add new link to print or download the registration sent page.

### PR DESCRIPTION
**Pull request by london-coder**
# DDCE-(StoryNumberHere)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: test/views/RegistrationSentViewSpec.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ class RegistrationSentViewSpec extends ViewBehaviours with ImplicitDateFormatter
 
   private val messageKeyPrefix = "registrationSent"
   private val section: Option[String] = Some(messages("declaration.section"))
-
+  private val firstLinkContent: String = "javascript:window.print()"
```
Comment (abhijeetardale) [inline test/views/RegistrationSentViewSpec.scala:28]: could you please remove this reference its not used inactula method pageWithPrintOrDownloadLink - url

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

File: test/views/RegistrationSentViewSpec.scala (hunk 1 above)
Comment (london-coder) [reply test/views/RegistrationSentViewSpec.scala:28]: hi, @abhijeetardale, do you want it removed, or add an assertion that checks the value?

Comment (london-coder) [reply test/views/RegistrationSentViewSpec.scala:28]: Assertion added.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -51,3 +51,8 @@ $hmrc-assets-path: "/register-charity-hmrc/assets/lib/hmrc-frontend/hmrc/";
 .hmrc-add-to-a-list__contents {
     padding-bottom: 35px;
 }
+
+.official-status-tag{
+ margin-bottom: -6px;
```
Comment (abhijeetardale) [inline app/assets/stylesheets/application.scss:56]: this changes are already in master, please rebase your branch

File: app/views/common/OfficialsSummaryView.scala.html [hunk 3]
```diff
@@ -73,7 +73,7 @@
                         </a>
                     </span>
                     <span class="hmrc-add-to-a-list__remove">
-                        <strong class="@getTagClass(official.isCompleted)">@messages(if(official.isCompleted){"index.section.completed"} else {"index.section.inProgress"})</strong>
+                        <strong class="@getTagClass(official.isCompleted) official-status-tag">@messages(if(official.isCompleted){"index.section.completed"} else {"index.section.inProgress"})</strong>
```
Comment (abhijeetardale) [inline app/views/common/OfficialsSummaryView.scala.html:76]: same as above

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

**Review by abhijeetardale — APPROVED** [review]

**Pull request merged**